scripts/global.lua

```diff
@@ -542,13 +542,11 @@ end
 
 
 function get_quest_info()
-   if LOC_add_quest_item then
-      LOC_add_quest_item()
+   if LOC_get_quest_info then
+      LOC_get_quest_info()
    end
-
-   add_quest_item(_"About...", _"This doesn't do much yet")
-   add_quest_item(_"Test1",    _"Some test info")
-   add_quest_item(_"Sensar",   _"He rages!")
+   -- Add global quest items here:
+   -- add_quest_item("key", "text")
```

Comment:
Did the format change? I uncommented this out and it gives a seg fault when you go to the quest menu. It seems to crash on the layout menu.cpp:395

Reply:
I couldn't reproduce this. Did you just uncomment line 549, ie.
``` lua
function get_quest_info()
   if LOC_get_quest_info then
      LOC_get_quest_info()
   end
   -- Add global quest items here:
   add_quest_item("key", "text")
end
```
Can you give more info on the seg fault?